lib/utils.js

```diff
@@ -19,4 +20,35 @@ exports.md5 = function(str) {
 exports.url = function(url){
   if (~url.indexOf('://')) return url;
   return 'http://' + url;
-};
\ No newline at end of file
+};
+
+/**
+ * Delete all files in [dirPath] older than [ttl].  Does not modify subdirectories.
+ */
+
+exports.eraseOldFiles = function(dirPath, ttl) {
+  ttl = typeof ttl !== undefined ? ttl : 0;
```

Comment:
ha. Weird I almost always do the check your way too, not sure why I overcomplicated it here.  :)